core/templates/dev/head/pages/about/About.js

```diff
@@ -33,6 +72,12 @@ oppia.controller('About', [
                               'Robyn Choo', 'Tricia Ngoon',
                               'Vikrant Nanda', 'Vinamrata Singal',
                               'Yarin Feigenbaum'];
+
+    $scope.onAboutOppiaClick = function(pageName) {
```

Comment:
Perhaps onTabClick(tabName)? All of these are on the same page and I think the name of the controller already gives the context that this is in the "About Oppia" page.